### DISCUSSION TITLE
How to have multiple Media Library?

### DISCUSSION THREAD

**Discussion by tangjh** [awcodes/filament-curator#439] [Q&A]
I want to have multiple media library filament resource each default to a specific directory.
eg Survey Media Library Resource will be pointing to directory /survey for all the media while Marketing Media Library Resource will be pointing to directory /marketing. 

Is it possible and how can I achieve this without duplicating the whole resource and multiple upload component?

**Comment by awcodes** [comment]
Best you're going to be able to do is to use the ->directory() and ->limitToDirectory() modifiers to control what is shown in the panel when using the picker.

The plugin is not setup to work with multiple "Libraries".
